src/common/GlobalStyles.js

```diff
@@ -4,8 +4,16 @@ export default createGlobalStyle`
   * {
     box-sizing: border-box;
   }
-
+:root {
+  --main-bg-color: #F77D33;
+}
   body {
-    margin: 0;
+    margin: 0 auto;
+    max-width: 1200px;
+    box-shadow: 0px 0px 12px 0px #bbb;
```

Comment:
The app uses height: 100vh, so the shadow is nerver visible, right?

Reply:
Yes, regarding the top and bottom border, I intended this shadow for left and right border on desktop view with a resolution of +1200px, but this is just a quick design fix. Desktop design can surely be improved later on.